apiserver/params/params.go

```diff
@@ -529,6 +529,7 @@ type LoginRequest struct {
 	Credentials string           `json:"credentials"`
 	Nonce       string           `json:"nonce"`
 	Macaroons   []macaroon.Slice `json:"macaroons"`
+	CLIArgs     string           `json:"cli-args"`
```

Comment:
Why a string and not a slice?
I guess it's just to record what was done rather than being used anywhere?

Reply:
Yes, that's right - it's just being captured so it can be logged, we don't need any structure.